src/api/mvc/variants/main.go

```diff
@@ -281,14 +201,7 @@ func VariantsIngest(c echo.Context) error {
 				// ---	 open vcf.gz
 
 				fmt.Printf("Opening %s !\n", gzippedFileName)
-				var separator string
-				if strings.HasPrefix(gzippedFileName, "/") {
-					separator = ""
-				} else {
-					separator = "/"
-				}
-
-				gzippedFilePath := fmt.Sprintf("%s%s%s", vcfPath, separator, gzippedFileName)
+				gzippedFilePath := gzippedFileName
```

Comment:
I know we are trying to get rid of this whole codebase, but I think it'd be best not to have the same value stored in three diff variables in this function (gzippedFileName, gzippedFilePath, tmpDestinationFileName - it makes removal logic a bit harder to follow.